backend/metadata_writer/src/metadata_writer.py

```diff
@@ -393,6 +397,9 @@ def is_kfp_v2_pod(pod) -> bool:
 
                 pods_with_written_metadata[obj.metadata.name] = None
 
-        except Exception as e:
-            import traceback
-            print(traceback.format_exc())
+        # If the for loop ended, a server-side timeout occurred. Continue watching.
+        pass
+
+    except urllib3.exceptions.ReadTimeoutError as e:
+        # Client side timeout, continue watching.
+        pass
```

Comment:
loooks like we can replace pass with `continue`

Reply:
Done.